packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -159,7 +167,27 @@ export default declare<State>((api) => {
         },
       },
       ImportDeclaration(path, state) {
-        if (path.node.source.value !== COMPILED_MODULE) {
+        const userLandModule = path.node.source.value;
+
+        const isCompiledModule = this.compiledModuleOrigins.some((compiledModuleOrigin) => {
+          if (userLandModule === DEFAULT_COMPILED_MODULE) {
+            return true;
+          }
+
+          if (
```

Comment:
TODO: Add a unit test for the babel plugin. I just tested it as an integration test in the react package, but best to test out edge cases. Was a bit naughty.